lambda/action_network.py

```diff
@@ -0,0 +1,138 @@
+import logging
+import os
+from typing import Dict
+
+import requests
+from dateutil import parser
+
+log = logging.getLogger(__name__)
+
+
+def get_object_or_empty_dict(key: str, event: Dict) -> Dict:
+    location = event.get(key)
+    if not location:
+        location = {}
+    return location
+
+
+def make_location(event: Dict) -> str:
+    location = get_object_or_empty_dict('location', event)
+    if location.get('country', 'US') != 'US':
+        return '{city}, {country}'.format(
+            city=location.get('locality'), country=location.get('country'))
+    return location.get('postal_code', '')
+
+
+def get_event_name(event: Dict) -> str:
```

Comment:
`return event.get('name', event.get('title', '')`

Reply:
I did the extra check because this will not return title is name is '' or None.  

(This is also why I write tests that seemingly test built in functionality.  I'd rather test it explicitly than rely on my memory of language details like this.)

```
>>> d = {'a':None}
>>> print(d.get('a', 'b'))
None
```

In the above example, this works.  It returns b.

```
d.get('a','b') or 'b'
```

But I consider this error prone (you've got to remember to change 'b' in two places).  Ultimately, a small function is reusable, easier to read, and less error prone.